python/ThirteenTeV/Grav_Hadronizer_TuneEE5C_13TeV_madgraph_differentPDF_herwigpp_cff.py

```diff
@@ -0,0 +1,50 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.HerwigppDefaults_cfi import *
+from Configuration.Generator.HerwigppUE_EE_5C_cfi import *
+from Configuration.Generator.HerwigppPDF_CTEQ6_LO_cfi import *									# Import CTEQ6L PDF as shower pdf
+from Configuration.Generator.HerwigppPDF_NNPDF30_NLO_cfi import herwigppPDFSettingsBlock as herwigppHardPDFSettingsBlock 	# Import NNPDF30 NLO as PDF of the hard subprocess
+from Configuration.Generator.HerwigppEnergy_13TeV_cfi import *
+from Configuration.Generator.HerwigppLHEFile_cfi import *
+from Configuration.Generator.HerwigppMECorrections_cfi import *
+
+# Showering LO MadGraph5_aMC@NLO LHE files with a different PDF for the hard subprocess 
+############ WARNING ######
+# This option should only be used with LO MadGraph5_aMC@NLO LHE files.
+# In case of NLO, MC@NLO matched LHE files this results most likely in a mismatch of phase space
+############ WARNING ######
+herwigNewPhysicsBlock = cms.PSet(
+    herwigNewPhysics = cms.vstring(
+        'cd /Herwig/Particles',
+        'create ThePEG::ParticleData graviton',
+        'setup graviton 39 graviton 1000 0.0 0.0 0.0 0 0 5 0',
+        'cd /'
+        ),
+    )
+
+generator = cms.EDFilter("ThePEGHadronizerFilter",
+	herwigDefaultsBlock,
+        herwigNewPhysicsBlock,
+	herwigppUESettingsBlock,
+	herwigppPDFSettingsBlock,
+	herwigppHardPDFSettingsBlock,			# Implementing renamed NNPDF30 config block
```

Comment:
Yes

BTW There should be no need to change herwigppPDFSettingsBlock to herwigppHardPDFSettingsBlock since both blocks (CTEQ6 and NNPDF30) should be automatically merged together.